.github/workflows/build-ios-beta.yml

```diff
@@ -22,7 +22,7 @@ jobs:
         uses: ./.github/actions/test
 
   build-ios-simulator:
-    runs-on: macos-12-large
+    runs-on: macos-13-large
```

Comment:
why not 14?

Reply:
I was unsure whether or not it was best practice for iOS to build on the oldest supported (e.g. like [appimage suggests](https://docs.appimage.org/reference/best-practices.html#binaries-compiled-on-old-enough-base-system)).
If that's ok, I'll go ahead and target MacOS-14 directly then :+1: I pushed the change